examples/manipulating_visualizing/plot_prob_atlas.py

```diff
@@ -0,0 +1,43 @@
+"""
+Plotting 4D Atlas maps or regions onto the anatomical image
+===========================================================
+
+We show here how to plot the atlas maps or statistical maps which are
+4D using a three different display types, for example using
+1. "contours", plots can be seen only in contours
+2. "filled_contours", plots can be seen in contours with fillings
+3. "continuous or overlays", plots can be seen in continuous overlays
+
+You can plot each map with each different color which will chose
```

Comment:
The colors "will be picked randomly in the given colormap". @banilo, could you review the documentation part of this PR at least? You're our expert!